modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/catalog/CPCatalogEntry.java

```diff
@@ -24,6 +26,8 @@ public interface CPCatalogEntry {
 
 	public long getCPDefinitionId();
 
+	public List<CPDefinitionOptionRel> getCPDefinitionOptionRels();
```

Comment:
why did you add this? it look it's never used

Reply:
it is an acceptance criteria to create a getCPDefinitionOptionRels method in the CPCatalogEntry class